### DISCUSSION TITLE
Issues with teacher enrolment

### DISCUSSION THREAD

**Discussion by NiccoloGranieri** [community/community#57528] [GitHub Education] [answered]
### Hi everyone,

I'm trying to verify my teacher status as I've done for 3 time already, but now that I'm at University of Chichester, Github won't let me use my institutional email @chi.ac.uk because it says that:

_We require applicants of University of Chichester to use one of these school-issued email addresses to apply:_

_[chialumni.ac.uk](http://chialumni.ac.uk/)_
_[stu.chi.ac.uk](http://stu.chi.ac.uk/)_
_Please select a different email address._

However, those emails are for students and alumni, not staff, teachers and lecturers.

Could you help me verify my account?

Thanks

**Comment by umichict** [comment]
I'm having a very similar issue. I'm on sabbatical at a different university and the university names are not aligned.

**Comment by chriskthomas** [comment]
I had a similar issue (school changed domains). I just typed the school name in the field instead of selecting the school. This did not set any email restrictions.

**Comment by NiccoloGranieri** [comment, marked as answer]
> I had a similar issue (school changed domains). I just typed the school name in the field instead of selecting the school. This did not set any email restrictions.

This worked! Thanks @chriskthomas, I don't know why your comment was marked as off-topic, thanks!

**Reply by chriskthomas** [reply]
They temporary banned me for answering too many questions :disappointed:. They really need to work on their moderation.

**Comment by pconesa** [comment]
I'm a researcher at cnb.csic.es and form does not work.  Keeps resetting after submitting the form.

https://education.github.com/discount_requests/application (as teacher)
